server.js

```diff
@@ -0,0 +1,50 @@
+
+
+//==============================================================================
```

Comment:
I love how well organized you have this server through your use of comments. Looks great!